clamd/clamav-daemon.service.in

```diff
@@ -11,6 +11,19 @@ ExecStart=@prefix@/sbin/clamd --foreground=true
 # Reload the database
 ExecReload=/bin/kill -USR2 $MAINPID
 TimeoutStartSec=420
+# Security options
+# Remove ProtectSystem, ProtectHome, and ReadWritePaths..
+# if you want clamav to be able to remove infected files.
+ProtectSystem=strict
+ProtectHome=read-only
+ReadWritePaths=/var/log/
+NoExecPaths=/
```

Comment:
Do you think we can whitelist /usr/local/bin/send_sms instead? I can add a comment saying "For added security, if you don't intend to use VirusEvent, please set this to `/`"

Reply:
> Do you think we can whitelist /usr/local/bin/send_sms instead? I can add a comment saying "For added security, if you don't intend to use VirusEvent, please set this to `/`"

`send_sms` is just an example.  We have no idea what admins will actually want to use for their VirusEvent commands. 

The instructions you provide seem like the best option.  Great idea!

Reply:
I have manually transplanted the clamd/clamav-daemon.service.in hardening settings to the clamav-daemon.service file of Arch Linux. `@CMAKE_INSTALL_FULL_LIBDIR@` replaced with `/usr/lib`. Though I had to set `PrivateTmp=yes` to `no` because this box is running a service under a different user that puts files to be scanned in the /tmp folder. The result is "7.8 EXPOSED". There is some more work to be done here.

Reply:
Setting `DynamicUser=yes` results in "`Can't save PID to file /run/clamav/clamd.pid: Permission denied`"

Reply:
Setting `CapabilityBoundingSet=` (drop all) results in `ERROR: Can't open /var/log/clamav/clamd.log in append mode (check permissions!)`

Reply:
For the` CapabilityBoundingSet` I ended up with `CapabilityBoundingSet=CAP_CHOWN CAP_SETGID CAP_SETUID CAP_DAC_OVERRIDE`.